data/changelog.txt

```diff
@@ -1,5 +1,22 @@
 # 0.F (2021-06-19)
 
+## Highlights
+Nested Containers rationalize inventory management and enable dropping and retrieving go-bags during fights.
+Achievements track your deeds and misdeeds across games.
+Proficiencies better represent deeper knowledge required for various endeavors, mostly crafting.
+Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment system.
```

Comment:
```suggestion
Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment systems.
```

Reply:
Would it be different systems or one system?

Reply:
Would it be different systems or one system?

Reply:
Need to drop the `a` if it's systems.
```suggestion
Bleeding added to both the player and monsters as the first step toward more comprehensive wound and wound treatment systems.
```

Reply:
It's singular, "wound and wound treatment" "system".